third_party/amd/lib/TritonAMDGPUToLLVM/Utility.h

```diff
@@ -106,6 +106,17 @@ bool isChainDotHead(mlir::triton::DotOpInterface dotOp);
 // Check if the opA of this tl.dot is the result of another tl.dot
 // in the same region
 bool isChainDotTail(mlir::triton::DotOpInterface dotOp);
+
+// LLVM is unable to recognize the dependency between AsyncCopy and
```

Comment:
Can you revise the comment to be more clear regarding 1) overall, how many async scopes we have, and how they are used to handle alias/non-alias by putting what ops in which scope, 2) then specifically for each of the following function what scopes we are attaching the given parameter to.